src/main/assembly/bin.xml

```diff
@@ -41,6 +41,8 @@
             <outputDirectory>bin</outputDirectory>
             <includes>
                 <include>easy-bag-index</include>
+                <include>easy-bag-index-service-start</include>
+                <include>easy-bag-index-initd.sh</include>
```

Comment:
I don't think these lines should be here. @janvanmansum thoughts? Not sure if they were added to other projects as well?

Reply:
They do not refer to any existing scripts, so obviously they should be removed. This is probably a left-over from the initd-support.